README.md

```diff
@@ -1,2 +1,24 @@
-# hello-rails-react
-Generate random greetings message
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
```

Comment:
- [ ] Let's customize readme file to much what you did in the project. you can follow microverse [template](https://github.com/microverseinc/readme-template).